Update dotnet-framework.md (#11614)
DataDog/documentation: content/en/tracing/setup_overview/setup/dotnet-framework.md

<div class="alert alert-warning">
<strong>Note:</strong> To set environment variables for a Windows Service, use the multi-string key <code>HKLM\System\CurrentControlSet\Services\{service name}\Environment</code> in the Windows Registry, as described above.
<br />
<br />

> IIS runs all applications under one Windows Service so it's best to use one of the other configuration options (Code, web.config, JSON file) for application specific settings like DD_SERVICE.

Reviewer: ```suggestion
Because IIS runs all applications under one Windows Service, Datadog recommends you use another configuration option (Code, web.config, or JSON file) for application-specific settings such as <code>DD_SERVICE</code>.
```
